configure.in

```diff
@@ -4,7 +4,35 @@ AC_CONFIG_AUX_DIR([./buildutils])
 AC_CONFIG_MACRO_DIR([./buildutils])
 AC_CANONICAL_TARGET
 AC_CONFIG_SRCDIR([include/exc_platform.h])
-AC_CONFIG_HEADER([include/config.h Driver/atool/src/include/nata/nata_config.h])
+AC_CONFIG_HEADER([include/config.h Driver/atool/src/include/nata/nata_config.h ])
+
+#--------------------------------------------------------------------
+# Bash function for trimming strings
+trim ()
+{
+    local var="$*"
+    # remove leading whitespace characters
+    var="${var#"${var%%[![:space:]]*}"}"
+    # remove trailing whitespace characters
+    var="${var%"${var##*[![:space:]]}"}"
+    printf '%s' "$var"
+}
+#--------------------------------------------------------------------
+
+AC_ARG_WITH(version-tag,
```

Comment:
_version_tag_ not version-tag?

Reply:
Nah. sorry, they are interchangeable since - is transformed to _

Reply:
No, actually should be "version-tag" cause this is a text of cmdline option. However, the variable which will be created to hold the value afterwards will be named "version_tag".